iphone/Classes/TiMediaAudioPlayerProxy.m

```diff
@@ -50,15 +50,19 @@ -(void)_listenerAdded:(NSString *)type count:(int)count
 {
 	if (count == 1 && [type isEqualToString:@"progress"])
 	{
-		progress = YES;
+		timer = [[NSTimer scheduledTimerWithTimeInterval:1 target:self selector:@selector(updateProgress:) userInfo:nil repeats:YES] retain];
```

Comment:
This will cause problems when the user creates different `progress` event listener. That's why it was in the initial creation only. Please test different event listeners and try to come up with a solution that can handle different event listeners as well (like before).